opensrp-app/src/main/java/org/smartregister/view/activity/DrishtiApplication.java

```diff
@@ -100,7 +100,7 @@ protected void attachBaseContext(android.content.Context base) {
         MultiDex.install(this);
     }
 
-    public Repository getRepository() {
+    public synchronized Repository getRepository() {
```

Comment:
since this will be returning a singleton is there a reason for changing method to `synchronized`